freqtrade/commands/build_config_commands.py

```diff
@@ -196,7 +206,11 @@ def ask_user_config() -> Dict[str, Any]:
     if not answers:
         # Interrupted questionary sessions return an empty dict.
         raise OperationalException("User interrupted interactive questions.")
-
+    answers["margin_mode"] = (
+        '\n    "margin_mode": "isolated",'
+        if answers.get("trading_mode") == 'futures'
+        else ''
+    )
```

Comment:
are we sure we want `margin_mode` to be missing from the configuration file if spot is selected?

I'd still provide it (but empty, maybe?) - which will make it easier on an existing config to switch to futures later on.

Reply:
Sure, I kind of just thought your didn't want it https://github.com/freqtrade/freqtrade/pull/6333#discussion_r795740253

Reply:
hah no - that comment was against hard-coding 'spot' and "null" for it without asking the user for "what do you want".
There's nothing wrong with having the key there - but the pr name ("new-config") suggested that we'd be asking users "you wanna trade futures?" - and then configure it properly (which this PR didn't do at the time i wrote this comment).